pkgs/tools/networking/pihole-ftl/default.nix

```diff
@@ -0,0 +1,34 @@
+{ cmake, gmp, fetchFromGitHub, libidn, nettle, readline, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "pihole-ftl";
+  version = "v5.3.4";
+
+  src = fetchFromGitHub {
+    owner = "pi-hole";
+    repo = "FTL";
+    rev = version;
+    sha256 = "00lzsa91p7557lyvy2avihlmqrbqcifyqqs9p9x9ixp034pgjpi2";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ gmp libidn nettle readline ];
+
+  patches = [ ./cmake.patch ./version.patch ];
+
+  installPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
I've moved the `patches` to `substituteInPlace` during `postPatch` phase, but I think this is the correct phase for these steps?

Reply:
Yes.